src/dowel/__init__.py

```diff
@@ -20,8 +19,7 @@
     'TextOutput',
     'LogOutput',
     'LoggerWarning',
-    'TabularInput',
+    'Tabular',
```

Comment:
this is now a private API, so it should not be in `__all__`, which is only for things people should be importing from your package.